docs/GettingStarted.md

```diff
@@ -0,0 +1,76 @@
+# Getting started
+
+All you'll need is a [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and the Hail source code. To clone the [Hail repository](https://github.com/broadinstitute/hail) using [Git](https://git-scm.com/), run
+```
+$ git clone https://github.com/broadinstitute/hail.git
+$ cd hail
+```
+The following commands are relative to the `hail` directory.
+
+You can also download the source code directly from [here](https://github.com/broadinstitute/hail/archive/master.zip).
+
+## Building and running Hail
+
+Hail may be built to run locally or on a Spark cluster. Running locally is useful for getting started, analyzing or experimenting with small datasets, and Hail development.
+
+### Locally
+The single command
+```
+$ ./gradlew installDist
+```
+installs Hail at `build/install/hail`. The initial build takes time as the build tool [Gradle](https://gradle.org/) installs and then downloads all Hail dependencies. The executable is `build/install/hail/bin/hail` (to run using `hail` add this `bin` to your path).
+
+Here are a few simple things to try. To list all commands, run
+```
+$ ./build/install/hail/bin/hail -h
+```
+To convert the included `sample.vcf` to Hail's `.vds` format, run
+```
+$ ./build/install/hail/bin/hail importvcf src/test/resources/sample.vcf write -o ~/sample.vds
+```
+Then to count the number of samples and variants, run
+```
+$ ./build/install/hail/bin/hail read -i ~/sample.vds count
+```
+To compute and output sample and variant quality control statistics, run
+```
+$ ./build/install/hail/bin/hail read -i ~/sample.vds variantqc -o ~/variantqc.tsv sampleqc -o ~/sampleqc.tsv
+```
+Note that during each run Hail writes a `hail.log` file in the current directory; this is useful to developers for debugging.
+
+### Spark cluster
+
+Hail is compatible with Spark versions 1.5 and 1.6, and uses 1.6.2 by default. Run
+```
+$ ./gradlew shadowJar -Dspark.version=VERSION
+```
+using the version of Spark on your cluster. The resulting JAR `build/libs/hail-all-spark.jar` can be submitted using `spark-submit`. See [Spark documentation](http://spark.apache.org/docs/1.6.2/cluster-overview.html) for details.
+
+### BLAS and LAPACK
+
+Hail uses BLAS and LAPACK-optimized linear algebra libraries. On Linux, these must be explicitly installed. On Ubuntu 14.04, run
+```
+$ apt-get install libatlas-base-dev
+```
+If natives are not found, `hail.log` will contain the warnings
+```
+Failed to load implementation from: com.github.fommil.netlib.NativeSystemLAPACK
+Failed to load implementation from: com.github.fommil.netlib.NativeSystemBLAS
+```
+See [netlib-java](http://github.com/fommil/netlib-java) for more information.
+
+## Testing Hail
```

Comment:
I like the sound of "Running the Tests" better.  Presumably it is already tested.  The user is just testing their setup/installation.